enjoy/src/renderer/components/widgets/translate-widget.tsx

```diff
@@ -59,7 +59,7 @@ export const TranslateWidget = () => {
         className="w-full p-0 z-50 select-text"
         updatePositionStrategy="always"
       >
-        <ScrollArea className="py-2 w-96 min-h-36 max-h-96 relative">
+        <ScrollArea className="py-2 w-96 min-h-36 max-h-96 overflow-y-auto relative">
```

Comment:
The `ScrollArea` component is intended to be scrollable, but it appears to only function correctly with a fixed height. Consider changing `min-h-36 max-h-96` to `h-96`.

Reply:
You are right, if your approach is adopted, scrolling can indeed be achieved when there is too much text. However, there are two small issues: 1. When the content is not enough, there will be a lot of blank space due to the fixed height, which will cover the page. 2. The scrolling container is the translated content rather than the entire widget.

Reply:
If you use `overflow-y-auto`, the `ScrollArea` component becomes unnecessary. You might as well use a `div` instead.

Reply:
I believe so. I have opted to use the `div` element instead of `ScrollArea` for clarity purposes.